registry-client.js

```diff
@@ -125,16 +125,16 @@ this.Registry = (function (Promise, XMLHttpRequest, DOMParser, undefined) {
 
               reject(new Error(`No service URL found for \nResource: ${resourceURI}\nStandard: ${standardURI}\nInterface: ${_interfaceURI}\nAuthType: ${authType}`))
             })
-            .catch(function (err) {
-              console.error(
-                'Error obtaining Service URL > ' + (err.error ? err.error : err)
-              )
+            .catch(function (request) {
+              var errorMsg = 'Could not obtain service URL for URI: ' + resourceURI + ' (' + request.status + ' - ' + request.statusText + ')'
+              reject(new Error(errorMsg))
```

Comment:
The `reject` call needs to be the last thing.  Put the console.error above this.